File: DropZone/DropZone.Core/Models/Parachute/ParachuteModel.cs
using System;

namespace DropZone.Core.Models
{
    public class ParachuteModel
    {
        public long Id { get; set; }

        public double Area { get; set; }

        public double LayingVolume { get; set; }

        public double Weight { get; set; }

        public int LayingCount { get; set; }

        public int SectionCount { get; set; }

        public bool IsReserve { get; set; }

        public DateTime? LayingDate { get; set; }

        public DateTime? MaintenanceDate { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long ManufacturerId { get; set; }

        public long HashBlockId { get; set; }


        public ManufacturerModel Manufacturer { get; set; }

        public HashBlockModel HashBlock { get; set; }
    }
}

File: DropZone/DropZone.Web/src/models/parachutes/parachute.model.ts
import { ManufacturerModel } from "../manufacturers/manufacturer.model";
import { HashBlockModel } from "../hashBlocks/hashBlock.model";

export class ParachuteModel {
	public Id: number;
	public Area: number;
	public LayingVolume: number;
	public Weight: number;
	public LayingCount: number;
	public SectionCount: number;
	public IsReserve: boolean;
	public LayingDate: Date;
	public MaintenanceDate: Date;
	public ModifiedAt: Date;

	public ManufacturerId: number;
	public HashBlockId: number;
	
	public Manufacturer: ManufacturerModel;
	public HashBlock: HashBlockModel;
};